crates/starknet-server/src/api/http/endpoints/blocks.rs

```diff
@@ -1,11 +1,39 @@
-use axum::Json;
+use axum::{Extension, Json};
 
 use crate::api::http::error::HttpApiError;
-use crate::api::http::models::{AbortedBlocks, AbortingBlocks, CreatedBlock};
-use crate::api::http::HttpApiResult;
+use crate::api::http::models::{AbortedBlocks, AbortingBlocks, BlockInfo};
+use crate::api::http::{HttpApiHandler, HttpApiResult};
 
-pub(crate) async fn create_block() -> HttpApiResult<Json<CreatedBlock>> {
-    Err(HttpApiError::GeneralError)
+pub(crate) async fn create_block(
+    Extension(state): Extension<HttpApiHandler>,
+) -> HttpApiResult<Json<BlockInfo>> {
+    let mut starknet = state.api.starknet.write().await;
+    starknet.update_pending_block_timestamp();
+    let _ = starknet.create_block().map_err(|_| HttpApiError::CreateEmptyBlockError);
```

Comment:
the error is not propagated.
```suggestion
    starknet.create_block().map_err(|_| HttpApiError::CreateEmptyBlockError)?;
```

Reply:
Fixed in d18ddf7bcf3740b4842af8b178833ffee7453caf